plugins/backup/fresh-backups

```diff
@@ -76,19 +76,22 @@ graph_title Fresh backups
 graph_info Number of fresh (<=${LIFETIME}d) backups archives in ${BACKUP_DIR}
 graph_args -l 0
 graph_category backup
-age0.label today
-age0.draw AREA
-age0.colour ${COLOUR[0]}
-age0.critical :
-age0.warning :
 EOF
-for AGE in $(seq 1 "${LIFETIME}"); do
-	cat << EOF
-age${AGE}.label older than $((AGE*24))h
-age${AGE}.draw STACK
+for AGE in $(seq "${LIFETIME}" -1 0); do
+	if [ "${AGE}" = 0 ]; then
+		echo "age${AGE}.label today"
+	else
+		echo "age${AGE}.label older than $((AGE*24))h"
+	fi
+	if [ "${AGE}" = "${LIFETIME}" ]; then
+		echo "age${AGE}.draw AREA"
+	else
+		echo "age${AGE}.draw STACK"
+	fi
+		cat << EOF
 age${AGE}.colour ${COLOUR[$AGE]}
-age${AGE}.warning :
-age${AGE}.critical :
+age${AGE}.warning 
```

Comment:
Do these two lines serve a purpose?
How about using `print_warning` instead?

Reply:
In my early experiment with this, the warning and critical values seemed to be inherited from those for `freshcount` when plotted. This forces no limit to be checked on those series.

`plugin.sh` is not safe to use with `-u` as it doesn't default undefined values (such as warnings and critical limits) to sane values.

Reply:
Inheritance: interesting. I am not sure, that this is the way it should work ...

`set -u`: indeed, this needs to be fixed!